### ISSUE TITLE
Support vim behaviour

### ISSUE THREAD

**Issue by user471** [Quicksaver/FindBar-Tweak#214]
I try to use this plugin with vimperator. Everything is fine but there is one problem. In vim you press search hotkey to open find bar then you type the word then you press Enter to start search and close the find bar. But with this plugin you need to press Esc or Ctrl+F to close the find bar. Is there any chance to support closing find bar by pressing Enter?

**Comment by Quicksaver** [comment]
Hmm, in theory this is an interesting feature. But can't you enable the option "Close the findbar when it loses focus" or "Always use the quick findbar" to achieve a similar result?

Granted, it doesn't close immediately when pressing enter. But it also doesn't require you to do any specific action to close it either.

**Comment by user471** [comment]
>"Close the findbar when it loses focus"

Yes, I already use it, so I can close findbar by pressing Esc. You still need to press something to lose focus. It could be solution if somehow when you press Enter findbar will lose focus.

>"Always use the quick findbar"

Even with "autoclose after 1 second" you need to wait when it will close. Pressing Enter is much faster.

Quick findbar has similar purpose. So I think the option like "Close Quick Findbar by Enter" could be a solution. Instead of waiting for autoclose you can just press Enter.

**Comment by Quicksaver** [comment]
Well I am planning some extra customization options for the enter key when typing in the findbar. Although this wasn't one of them, it shouldn't be too hard to add it as well.

**Comment by user471** [comment]
thanks
